File: Python/Card.py
from payment import Payment
from datetime import date
class Card(Payment):
    Frachise = str
    Duedate = date.today()
    CVV = str

    def __init__(self, id, Franchise, Duedate, CVV):
        super.__init__(id)
        self.Franchise = Franchise
        self.Duedate = Duedate
        self.CVV = CVV

File: Java/Card.java
package Java;
import java.util.Date;

class Card extends Payment {
    String Frachise;
    Date Duedate;
    String CVV;

    public Card(Integer id, String Franchise, Date Duedate, String CVV){
        super(id);
        this.Frachise = Franchise;
        this.Duedate = Duedate;
        this.CVV = CVV;
        

        this.printlnDataPayment();
    }
}

